zio-cli/shared/src/main/scala/zio/cli/Options.scala

```diff
@@ -331,6 +343,8 @@ object Options extends OptionsPlatformSpecific {
     override def isValid(input: String, conf: CliConfig): IO[ValidationError, List[String]] = for {
       _ <- validate(List(self.names.head, input), conf)
     } yield List(self.names.head, input)
+
+    private def placeholder: String = "<" + self.pseudoName.getOrElse(self.primType.typeName) + ">"
```

Comment:
Optional: Consider removing `<>`. If user wants to have it - he can specify it as part of placeholder. With `<>` here there is no way to create a placeholder without `<>`

Reply:
It was already like that for `Args` so I wanted it to be consistent.